.github/workflows/pull-request-lint.yml

```
@@ -27,6 +27,7 @@ jobs:
             examples
             build
             repo
+            playground
```

While you are at it:

```suggestion
            vscode
            playground
```